CHANGELOG.md

```diff
@@ -5,9 +5,11 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 ## Unreleased
 
 ### Added
-
 - The new ARG `EARTHLY_GIT_REFS` will contain the references to the current git commit, this ARG must be enabled with the `VERSION --git-refs` feature flag. [#3341](https://github.com/earthly/earthly/pull/3341)
 
+### Fixed
+- `prune --age` does not support `d` (for days) as a suffix contradictory to help message [#3401](https://github.com/earthly/earthly/issues/3401)  
```

Comment:
I had to read this a few times before I understood what it meant, maybe something like this instead?
```suggestion
- `prune --age` did not support `d` (for days) suffix, even thought `earthly --help` said it did [#3401](https://github.com/earthly/earthly/issues/3401)  
```